recipes-bsp/alsa-state/alsa-state.bbappend

```diff
@@ -1,4 +1,6 @@
 # Prepend path to override files from upstream recipe
 FILESEXTRAPATHS:prepend:imx-generic-bsp := "${THISDIR}/${PN}/imx-generic-bsp:"
+FILESEXTRAPATHS:prepend:imx-nxp-bsp := "${THISDIR}/${PN}/imx-nxp-bsp:"
+FILESEXTRAPATHS:prepend:imx-mainline-bsp := "${THISDIR}/${PN}/imx-mainline-bsp:"
```

Comment:
I believe the way to fix this is to just drop the BSP overrides:

`FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"

Reply:
@thochstein You are right. It is much more simple to implement it. Let me update it